ecs-agent/csiclient/csi_client.go

```diff
@@ -18,13 +31,19 @@ const (
 	PROTOCOL = "unix"
 )
 
+// CSIClient is an interface that specifies all supported operations for Agent uses.
```

Comment:
It would be good if you can add a sentence about what the CSI driver is so it will be easier for reviewers without much context about the code to understand.

Reply:
Ack